web/frontend/src/lib/schedule.ts

```diff
@@ -0,0 +1,28 @@
+export const scheduleAsyncPoll = (callback: () => Promise<void>, interval: number) => {
+  let cancelled = false;
+  let timeoutId = -1;
+
+  const refreshAndScheduleNext = async () => {
+    await callback();
+
+    // eslint-disable-next-line no-use-before-define
```

Comment:
Happily enough, the common JS config will remove the need for this override by configuring the `no-use-before-define` rule to only complain about actual errors